src/app/components/RoundedBalance/index.tsx

```diff
@@ -43,6 +45,30 @@ export const RoundedBalance: FC<RoundedBalanceProps> = ({
       <PlaceholderLabel label={tickerName} />
     )
 
+  if (number.isGreaterThan(100_000) && roundLargeNumbers) {
+    return (
+      <Tooltip
+        arrow
+        placement="top"
+        title={t('common.valueInToken', { value: number.toFormat(), ticker: tickerName })}
+        enterDelay={tooltipDelay}
+        enterNextDelay={tooltipDelay}
+      >
+        <span>
+          {t('common.valuePair', {
+            value: number.toFixed(),
```

Comment:
Hm. How does this work :O this makes a string. I guess it silently does parseFloat?

Reply:
🤷  Output seems Ok when I look at token pages

Reply:
it can take strings :O https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/format#using_format_with_a_string